### PR TITLE
4 get api articles article id inc error handling

### PR DESCRIPTION
**Pull request by Antony-Lester**
there seems like there could be some merge issues regarding this pull request,

### CODE REVIEW

**Comment by Antony-Lester** [comment]
I think have corrected merge issues correctly now.

File: __tests__/app.test.js [hunk 1]
```diff
@@ -5,38 +5,83 @@ const seed = require('../db/seeds/seed');
...
+                        );
+                    });
+                });
+        });
+        test('status:404 Not Found', () => {
+            return request(app)
+                .get('/api/topic')
+                .expect(404)
+                .then(({ body }) => {
+                    expect(body.msg).toBe('Invalid path');
```
Comment (alexdevdavis) [inline __tests__/app.test.js:41]: 'not found' is typically used for 404s, but that's ultimately up to you :)

File: app.js [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
 }) 
-
-app.all('*', (req, res, next) => {
-    res.status(404)
-        .send("error: path not found")
-})
-
+//404
+app.use((err, req, res, next) => {
+    console.log("IN USE 2", err.code)
```
Comment (alexdevdavis) [inline app.js:23]: I assume you want this console.log to stay here for the time being

**Review by alexdevdavis — APPROVED** [review]
Happy to approve this PR. Great job!

File: __tests__/app.test.js (hunk 1 above)
Comment (Antony-Lester) [reply __tests__/app.test.js:41]: thanks for the advice!

**Pull request merged**